opm/io/eclipse/EGrid.cpp

```diff
@@ -93,11 +93,13 @@ EGrid::EGrid(const std::string &filename, std::string grid_name) :
                 nijk[0] = gridhead[1];
                 nijk[1] = gridhead[2];
                 nijk[2] = gridhead[3];
-                
-                numres = gridhead[24];
-               
-                if (gridhead.size() > 26)
+
+                numres == (gridhead.size() > 24)
+                    ? gridhead[24] : 1;
+
+                if (gridhead.size() > 26) {
```

Comment:
maybe
```c++
m_radioal = gridhead.size() > 26 ? gridhead[26] > 0 : false
```

Reply:
I ended up using `(size() > 26) && (value > 0)` as a compromise.

Reply:
fair enough, mostly the inconsistent ternary vs if usage I jerked on.